content/en/news/releases/1.15.x/announcing-1.15.2/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.15.1
+linktitle: 1.15.1
+subtitle: Patch Release
+description: Istio 1.15.1 patch release.
+publishdate: 2022-09-26
+release: 1.15.1
+---
+
+This release includes security fixes in Go 1.19.2 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what is different between Istio 1.15.1 and Istio 1.15.2.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0 and timeout is disabled. Previous behavior is using envoy's default value. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

Comment:
```suggestion
- **Fixed** an issue where the default `idleTimeout` for the passthrough cluster has been changed to `0s` since 1.14.0 and the timeout is disabled. Previous behavior was using Envoy's default value. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

I'm not really sure what this fix is about. We mention pre 1.14.0 we used Envoy's default value.  In 1.14.0 the default is 0 and the timeout is disabled. Now we fixed it, but what did we fix it to?

Reply:
If I understand the linked issue correctly, the default in Envoy is 1h and 0s disables the timeout... so we no longer use 0s as the timeout. I'm guessing, the release note is not clear.